sympy/integrals/rubi/utility_function.py

```diff
@@ -258,7 +261,7 @@ def IntegersQ(*var):
     return all(IntegerQ(i) for i in var)
 
 def ComplexNumberQ(*var):
-    """
+    r"""
```

Comment:
You haven't addressed this comment https://github.com/sympy/sympy/pull/13261#discussion_r155200701